ui/apps/platform/src/Containers/VulnMgmt/Reports/Form/CollectionSelection.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useMemo, useState, ReactElement, useCallback } from 'react';
+import {
+    Button,
+    ButtonVariant,
+    Flex,
+    FlexItem,
+    Select,
+    SelectOption,
+    SelectProps,
+    SelectVariant,
+} from '@patternfly/react-core';
+import sortBy from 'lodash/sortBy';
+import uniqBy from 'lodash/uniqBy';
+
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import { Collection, listCollections } from 'services/CollectionsService';
+import CollectionsFormModal from 'Containers/Collections/CollectionFormModal';
+import { useCollectionFormSubmission } from 'Containers/Collections/hooks/useCollectionFormSubmission';
+import { useFormik } from 'formik';
+import useSelectToggle from 'hooks/patternfly/useSelectToggle';
+import { usePaginatedQuery } from 'hooks/usePaginatedQuery';
+
+const COLLECTION_PAGE_SIZE = 10;
```

Comment:
Thoughts on a smaller page size? I feel like a page size of ~5 might make users more likely to attempt to search by name before scrolling down the infinite list.